inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/sink/TubeSink.java

```diff
@@ -121,6 +137,10 @@ public void configure(Context context) {
         // create producer holder
         producerHolder = new TubeProducerHolder(getName(),
                 usedMasterAddr, configManager.getMqClusterConfig());
+        // get statistic configure items
+        maxMonitorCnt = context.getInteger(MAX_MONITOR_CNT, 300000);
+        statIntervalSec = context.getInteger(STAT_INTERVAL_SEC, 60);
```

Comment:
make the configurations shown in config files.

Reply:
done